my-burger-app/src/components/burger.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react';
+
+import up from '../assets/up.png';
+import under from '../assets/under.png';
+import tomato from '../assets/tomato.png';
+import salad from '../assets/salad.png';
+import cheese from '../assets/cheese.png';
+import meat from '../assets/meat.png';
+import onion from '../assets/onion.png';
+import pickle from '../assets/pickle.png';
+
+const Burger = ({ ingredients, burgerState }) => {
+  const [ifIngredients, setIfIngredients] = useState(false);
+  const [totalPrice, setTotalPrise] = useState(1);
```

Comment:
Should the initial price be zero?

Reply:
The initial price is 1$ because we have a burger bun by default.